_data/people/masonproffitt.yml

```
@@ -74,3 +87,32 @@ presentations:
   location: Virtual
   focus-area: as
   project: uproot
+- title: "Data handling: uproot, awkward & vector"
+  date: 2021-11-03
+  url: https://indico.cern.ch/event/1076231/contributions/4560398/
+  video: https://indico.cern.ch/event/1076231/contributions/4560398/attachments/2338579/4017718/agc_uproot_awk.mp4
```

```suggestion
  video: https://indico.cern.ch/event/1076231/contributions/4560398/
```

I'm not sure if @henryiii wants the event URL vs. the artifact URL to apply to videos as well as slides, but I would assume it would.

---

This doesn't make sense to me. Why even use the `video` field in that case? It seems like it would be better to just remove the line.

---

This would still show the reader there's a video. We can keep it, but indico links are not very stable.

---

oh sorry @masonproffitt. I guess I missed this. Didn't mean to ignore you for 2 months. :grimacing: Thanks Henry!